specification/web/resource-manager/readme.md

```diff
@@ -262,6 +262,10 @@ directive:
 - suppress: R3026
   from: ContainerAppsRevisions.json
   reason: Patch operation will be implemented in later version.
+suppressions:
```

Comment:
>suppressions [](http://example.com/codeflow?start=0&length=12)

Can you add more context on this?

Reply:
These suppressions are made in relation to this: https://github.com/Azure/azure-rest-api-specs/pull/26545

We want to release version 2023-01-01 before this Friday Nov 10 so that it is included in the Python SDK release.

Reply:
@kamperiadis I see in the PR mentioned, the response is modified? Do you still need this suppression? Also why not making changes on the same PR, tracking them each will be difficult and cost more time.